### PR TITLE
Meeting summaries 2016-10-27

### PR DESCRIPTION
**Pull request by G1lius**

### CODE REVIEW

File: _posts/en/meetings/2016-10-27-meeting.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ version: 1
  
 ## Notes / short topics
 
-- 0.13.1 is released. ([Binaries](https://bitcoin.org/bin/bitcoin-core-0.13.1/), [magnet link](magnet:?xt=urn:btih:dbe48c446b1113890644bbef03e361269f69c49a&dn=bitcoin-core-0.13.1&tr=udp%3A%2F%2Ftracker.openbittorrent.com%3A80%2Fannounce&tr=udp%3A%2F%2Ftracker.publicbt.com%3A80%2Fannounce&tr=udp%3A%2F%2Ftracker.ccc.de%3A80%2Fannounce&tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969&tr=udp%3A%2F%2Ftracker.leechers-paradise.org%3A6969&ws=https%3A%2F%2Fbitcoin.org%2Fbin%2F), [mailing list post](https://lists.linuxfoundation.org/pipermail/bitcoin-core-dev/2016-October/000023.html), [blogpost](https://bitcoincore.org/en/2016/10/27/release-0.13.1/)) 
+- 0.13.1 is released. ([Binaries](https://bitcoin.org/bin/bitcoin-core-0.13.1/), [mailing list post](https://lists.linuxfoundation.org/pipermail/bitcoin-core-dev/2016-October/000023.html), [blogpost](https://bitcoincore.org/en/2016/10/27/release-0.13.1/))
```
Comment (btcdrak) [inline _posts/en/meetings/2016-10-27-meeting.md]: Why remove the magnet link?

**Comment by btcdrak** [comment]
merged in 1e9549ff83ad61b0827af54812cf86fece48f638

**Comment by G1lius** [comment]
link didn't seem to work when I previewed it.

Op 2-nov.-2016 10:08 schreef "฿tcDrak" <notifications@github.com>:

**Comment by btcdrak** [comment]
Maybe you dont have a Torrent app installed?

**Comment by G1lius** [comment]
I do, link works on the site.

I probably need to stop using the preview on github.com, not the first time things show up differently.

**Pull request closed without merging**